Port in-product tile instructions for some OAuth integrations to the README that gets parsed inside the "Configuration instructions" section (#23673)
DataDog/integrations-core: docusign/README.md

## Setup

> To collect Docusign logs, click the **Authorize** button to authenticate using OAuth.

Reviewer: ```suggestion
To collect Docusign logs, click **Authorize** to authenticate using OAuth.
```

- Docusign account with a plan that includes Docusign Monitor

### Generate API credentials in Docusign

1. Log in to [Developer Admin Console][2].
2. Get your **Account Name** from the **Account Profile** page.
3. Access the **Apps and Keys** page under the **Integrations** section.
4. Obtain the **User ID** from the **My Account Information** section.
5. Click on **Add App and Integration Key** button.
6. Provide an App Name and click **Create App**.
7. Get your **Integration Key** from the General Info section.
8. Click on **Generate RSA**, to get **RSA Private Key**.
9. Click on **ADD URI**, set Redirect URI to `http://localhost/`.
10. Click on **Save** button.
11. Perform application consent by opening the accompanying Sample URL after replacing the placeholders 
    - URL: `<BASE_URI>/oauth/auth?response_type=code&scope=signature impersonation organization_read&client_id=<YOUR_INTEGRATION_KEY>&redirect_uri=<YOUR_REDIRECT_URI>`
        - `<BASE_URI>`: Use `https://account-d.docusign.com` for Developer or `https://account.docusign.com` for Production.
        - `<YOUR_INTEGRATION_KEY>`: Replace with your Integration Key (Client ID) obtained previously.

>         - `<YOUR_REDIRECT_URI>`: Replace with your redirect URI, e.g., `http://localhost/`.

Author: I believe all of this is outdated since Docusign is an OAuth integration.

> In order to add a new Docusign account, you first have to make sure:

Reviewer: ```suggestion
To add a Docusign account, make sure that:
```

> - The account has **admin access permission** within your Docusign organization. Otherwise, we won't be able to collect log events for you.

Reviewer: ```suggestion
- The account has **admin access permission** within your Docusign organization. Otherwise, Datadog can't collect log events for you.
```

> - The account **may not be an admin** of more than one organization on Docusign as explained [here][2].

Reviewer: ```suggestion
- The account is **not an admin** of more than one Docusign organization, as explained in [How to get monitoring data][2].
```

> - **Docusign Monitor** feature is enabled. This integration is only available for accounts with the paid version of [Monitor][3].

Reviewer: ```suggestion
- The **Docusign Monitor** feature is enabled. This integration is only available for accounts with the paid version of [Monitor][3].
```
